pkg/download/retry.go

```diff
@@ -21,7 +22,7 @@ var (
 const (
 	// time to sleep between retries is an exponential backoff formula:
 	//   t(n) = k * m^n
-	expRetryN = 7 // how many times we retry the Download
+	expRetryN = 3 // how many times we retry the Download
```

Comment:
Please edit retry_test.go/TestWithRetries_failing_validateNumberOfCalls and retry_test.go/TestWithRetries_healingServer to reflect this change, right now both are failing on lines 56 and 82 respectively

Reply:
fixed the test